pkg/env/sensor.go

```diff
@@ -103,4 +103,10 @@ var (
 	// EnrichmentPurgerTickerCycle controls how frequently purger is run to check for collector updates
 	// (network flows & container endpoints) that stuck in Sensor's memory. Set to zero to completely disable the purger.
 	EnrichmentPurgerTickerCycle = registerDurationSetting("ROX_ENRICHMENT_PURGER_UPDATE_CYCLE", 30*time.Minute, WithDurationZeroAllowed())
+
+	// CollectorIServiceChannelBufferSize is used to specify the size of the queue for the collector internal service.
+	CollectorIServiceChannelBufferSize = RegisterIntegerSetting("ROX_SENSOR_COLLECTOR_CHANNEL_BUFFER_SIZE", 0)
+
+	// ProcessSignalChannelBufferSize is used to specify the size of the queue for process signals coming from collector.
+	ProcessSignalChannelBufferSize = RegisterIntegerSetting("ROX_SENSOR_SIGNAL_CHANNEL_BUFFER_SIZE", 0)
```

Comment:
Thanks for setting this to 0! I am a bit concerned as totally not-loaded Sensor is dropping messages when this is 0. I will check if making it 1 or 2 can remove the drop rate completely. If not, we should investigate, because something may be blocking/holding mutexes for too long.